src/python/magnum/math.vector.h

```diff
@@ -133,7 +133,7 @@ template<class T, class ...Args> void everyVectorBuffer(py::class_<T, Args...>&
         /* Buffer protocol. If not present, implicit conversion from numpy
            arrays of non-default types somehow doesn't work. There's also the
            other part in vectorBuffer(). */
-        .def(py::init([](py::buffer other) {
+        .def(py::init([](const py::buffer &other) {
```

Comment:
```suggestion
        .def(py::init([](const py::buffer& other) {
```

Clang Tidy is made out of flesh here, sorry :D